cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -209,21 +216,47 @@ def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
         casecomponent: Infra
         tags: reconfigure
         testSteps:
-            1. Change number of CPU sockets and amount of memory while VM is runnng.
+            1. Add and remove the disk while VM is running
             2. Go to Compute -> infrastructure -> Virtual Machines -> Select Vm
             3. Go to VM reconfiguration
-            4.Change number of CPU sockets and amount of memory, save and submit
+            4. Click on Add Disk -> select disk_type and disk_mode , save and submit
             5. Check the count in VM details page
+            6. Remove the disk and Check the count in VM details page
     """
-    pass
+    orig_config = small_vm.configuration.copy()
+    new_config = orig_config.copy()
+    new_config.add_disk(
+        size=500, size_unit='MB', type=disk_type, mode=disk_mode)
+
+    add_disk_request = small_vm.reconfigure(new_config)
+    # Add disk request verification
+    wait_for(add_disk_request.is_succeeded, timeout=360, delay=45,
```

Comment:
Question: Is there any harm in lowering the `delay` to e.g. 10-15 s?

Reply:
add 15